src/time.rs

```diff
@@ -24,7 +24,7 @@ pub(crate) fn mock_time(time: &PyAny) -> PyResult<()> {
     Ok(())
 }
 
-#[pyclass]
+#[pyclass(module = "parsec._parsec")]
```

Comment:
Why adding `module = "parsec._parsec"` ?

from the documentation:

| | |
| --- | --- |
| `module = "module_name"` | Python code will see the class as being defined in this module. Defaults to builtins. |

Reply:
This is needed by pickle to know where to look when deserializing this class

Reply:
Could you add a comment for that ?